homeassistant/components/zwave/cover.py

```diff
@@ -184,3 +204,105 @@ def close_cover(self, **kwargs):
     def open_cover(self, **kwargs):
         """Open the garage door."""
         self.values.primary.data = "Opened"
+
+
+class FibaroFGRM222(ZwaveRollershutter):
+    """Implementation of proprietary features for Fibaro FGR-222 / FGRM-222.
+
+    This adds support for the tile feature for the ventian blind mode.
+    For this to work, you need to set the Zwave device configuration
+    to veneian blind mode with reports type to Fibaro.
+
+    To enable this you need to configure the devices to use the venetian blind
```

Comment:
This paragraph is a more elaborate version of the second sentence of the previous paragraph. Maybe just delete the previous one?

Reply:
fixed